### PR TITLE
Dashboard contacts

### PR DESCRIPTION
**Pull request by danielrus**

### CODE REVIEW

File: src/app/dashboard/components/dashboard-contacts/dashboard-contacts.component.html [hunk 1]
```diff
@@ -1 +1,67 @@
-<p>dashboard-contacts works!</p>
+<h2>
+  <strong>{{ total }}</strong>
+  {{ 'strings.headingAllContacts' | translate }}
+</h2>
+
+<ul *ngIf="contacts" [ngClass]="{ summary: true, compact: diseases.length }">
+  <li class="contact-unconfirmed">
+    <span class="count">{{ contacts.UNCONFIRMED }}</span>
+    <span class="percentage">{{ contacts.UNCONFIRMED / total | percent }}</span>
```
Comment (adriandumavitagroup) [inline src/app/dashboard/components/dashboard-contacts/dashboard-contacts.component.html]: what if total = 0 ?

File: src/app/dashboard/components/dashboard-contacts/dashboard-contacts.component.html [hunk 2]
```diff
@@ -1 +1,67 @@
...
+    <mat-icon class="material-icons-round icon">gpp_bad</mat-icon>
+    <span class="info">{{ 'captions.dashboardUnconfirmedContact' | translate }}</span>
+    <mat-progress-bar [value]="(contacts.UNCONFIRMED / total) * 100"></mat-progress-bar>
+  </li>
+  <li class="contact-confirmed">
+    <span class="count">{{ contacts.CONFIRMED }}</span>
+    <span class="percentage">{{ contacts.CONFIRMED / total | percent }}</span>
+    <mat-icon class="material-icons-round icon">connect_without_contact</mat-icon>
+    <span class="info">{{ 'captions.dashboardConfirmedContact' | translate }}</span>
+    <mat-progress-bar [value]="(contacts.CONFIRMED / total) * 100"></mat-progress-bar>
```
Comment (adriandumavitagroup) [inline src/app/dashboard/components/dashboard-contacts/dashboard-contacts.component.html:19]: what if total = 0 ?

File: src/app/dashboard/components/dashboard-contacts/dashboard-contacts.component.html [hunk 3]
```diff
@@ -1 +1,67 @@
...
+  {{ 'strings.headingAllContacts' | translate }}
+</h2>
+
+<ul *ngIf="contacts" [ngClass]="{ summary: true, compact: diseases.length }">
+  <li class="contact-unconfirmed">
+    <span class="count">{{ contacts.UNCONFIRMED }}</span>
+    <span class="percentage">{{ contacts.UNCONFIRMED / total | percent }}</span>
+    <mat-icon class="material-icons-round icon">gpp_bad</mat-icon>
+    <span class="info">{{ 'captions.dashboardUnconfirmedContact' | translate }}</span>
+    <mat-progress-bar [value]="(contacts.UNCONFIRMED / total) * 100"></mat-progress-bar>
```
Comment (adriandumavitagroup) [inline src/app/dashboard/components/dashboard-contacts/dashboard-contacts.component.html:12]: what if total = 0 ?

File: src/app/dashboard/components/dashboard-contacts/dashboard-contacts.component.html [hunk 4]
```diff
@@ -1 +1,67 @@
...
+  </li>
+  <li class="contact-confirmed">
+    <span class="count">{{ contacts.CONFIRMED }}</span>
+    <span class="percentage">{{ contacts.CONFIRMED / total | percent }}</span>
```
Comment (adriandumavitagroup) [inline src/app/dashboard/components/dashboard-contacts/dashboard-contacts.component.html]: what if total = 0 ?

**Review by adriandumavitagroup — APPROVED** [review]

**Review by sandorbanvitagroup — APPROVED** [review]

**Pull request merged**